mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulHome.kt

```diff
@@ -48,7 +48,7 @@ fun StatefulHome(
         navController = controller,
         startDestination = SocialRoute,
     ) {
-      composable(SocialRoute) { StatefulFollowingScreen(user, Modifier.fillMaxSize()) }
+      composable(SocialRoute) { StatefulFollowingScreen(user, modifier = Modifier.fillMaxSize()) }
```

Comment:
Nitpick : To be consistent with codes below, remove the `modifier =`

```suggestion
      composable(SocialRoute) { StatefulFollowingScreen(user,  Modifier.fillMaxSize()) }
```

Reply:
Thanks for noting that, was looking for it after my change!